uint32 start_x
uint32 start_y
uint32 end_x
uint32 end_y

uint32 viewport_height
uint32 viewport_width

float32 focal_length

float32[16] projection_matrix
geometry_msgs/Pose camera_pose

bool is_deep_selection

int32[] polyline_x
int32[] polyline_y
